content/gocourse-2019-05-melbourne.slide

```diff
@@ -1142,6 +1142,35 @@ Rob Pike's Talk ([[https://blog.golang.org/toward-go2][Towards Go 2]])
 Dave Cheney Blog ([[https://dave.cheney.net/paste/the-past-present-and-future-of-go2.pdf][Past Present and Future of Go]])
 
 
+* Go for Java devs
+
+- Functions are lambdas
```

Comment:
What do you mean be that? Do you mean that anonymous function declarations are possible? Inline function declarations? that functions can be assigned to variables?
not really clear on this.
I know it might be a big ask, but maybe an example on goplayspace would go a long way with this?

Reply:
Java tends to force you to think about lambdas and methods as two separate objects (indeed they are in java, lambdas are instances of a class implementing the `Function` interface).
Will add some examples (inline, assign to variable, pass as argument).

Reply:
Examples added